scripts/iql-lesson-sync/README.md

```diff
@@ -0,0 +1,57 @@
+# Upload to IBM Quantum Learning
+
+This script uploads lessons to [IBM Quantum
+Learning](learning.quantum-computing.ibm.com).
+
+Lessons to be uploaded must be a directory containing a single notebook and any
+images it uses (the API can't handle image attachments inside the notebooks).
+
+```sh
+# Example
+lesson-root
+├── index.ipynb
+└── images
+    └── image-1.png
+```
+
+To sync your lessons automatically with the API, you'll need to add a
+`iql.conf.yaml` to the root of your content folder.
```

Comment:
`learning.conf.yaml`?

Reply:
This is a bit more complicated as these files already exist in internal repos. A future feature could be to search for both, but for now I might suggest a comment in the YAML file explaining what it is.

Reply:
Sounds great! Thanks.

Reply:
@frankharkins @Eric-Arellano be careful, `Platform` is a term that we don't want to associate with `Learning`, since Platform is another of the applications:

<img width="269" alt="Screenshot 2023-11-21 at 9 36 59 AM" src="https://github.com/Qiskit/documentation/assets/1007051/f5a4a8c1-52b6-4970-a674-d498622d9c5c">

Reply:
Good point. @frankharkins would it be possible to fix that ambiguity? Naming has tripped us up before, like my confusion yesterday with `url` vs `learn` in `toc.yaml`.